code_annotations/config_and_tools/config/feature_toggle_annotations.yaml

```diff
@@ -11,17 +11,17 @@ annotations:
   feature_toggle:
     - ".. toggle_name:":
     - ".. toggle_implementation:":
-        choices: [ExperimentWaffleFlag, WaffleFlag, WaffleSample, WaffleSwitch, CourseWaffleFlag, ConfigurationModel, DjangoSetting]
+        choices: [WaffleFlag, WaffleSwitch, CourseWaffleFlag, ExperimentWaffleFlag, ConfigurationModel, DjangoSetting]
     - ".. toggle_default:":
     - ".. toggle_description:":
-    - ".. toggle_category:":
     - ".. toggle_use_cases:":
-        choices: [incremental_release, launch_date, monitored_rollout, graceful_degradation, beta_testing, vip, opt_out, opt_in, open_edx]
+        choices: [temporary, circuit_breaker, vip, opt_out, opt_in, open_edx]
     - ".. toggle_creation_date:":
     - ".. toggle_expiration_date:":
     - ".. toggle_warnings:":
```

Comment:
Have we found `toggle_warnings` to be a useful field to prescribe?  Can any warnings be part of the `description` or the `how-to`?

Reply:
@regisb commented elsewhere that he liked how he could make it stick out in readthedocs. Not sure if we have an example.

Reply:
Yes, we use the `toggle_warnings` field to output a warning in the html docs, similar to the RST's `.. warning::` field. Here is an example output, documented in this PR:

https://github.com/edx/edx-platform/pull/24815#issuecomment-678096331
![docs-warnings](https://user-images.githubusercontent.com/44319/90865374-930c2b00-e392-11ea-9012-58baba4fc0ca.png)

Reply:
I think this is fine. It would be nice if the how_to includes a screenshot to clarify.

Reply:
Nice! I like how the warnings stand out.  As a toggle creator, I'd like to know what the end result will look like so I know what to enter in these fields.  When we're at the stage of meta-documenting the How-To of Toggle Annotations, we can provide this information.

Great work thus far!